src/mono/wasm/debugger/DebuggerTestSuite/EvaluateOnCallFrameTests.cs

```diff
@@ -542,6 +542,7 @@ async Task EvaluateOnCallFrameFail(string call_frame_id, params (string expressi
                     ("this.CallMethodWithParmBool(true)", TString("TRUE")),
                     ("this.CallMethodWithParmBool(false)", TString("FALSE")),
                     ("this.CallMethodWithParmString(\"concat\")", TString("str_const_concat")),
+                    ("this.CallMethodWithParmString(\"\\\"\\\"\")", TString("str_const_\"\"")),
```

Comment:
I couldn't help myself
```suggestion csharp
                    ("this.CallMethodWithParmString(\"\\\"\\\"\")", TString("str_const_\"\"")),
                    ("this.CallMethodWithParmString(\"🛶\")", TString("str_const_🛶")),
                    ("this.CallMethodWithParmString(\"\\uD83D\\uDE80\")", TString("str_const_🛶")),
```
fails in the best possible way
```
[[this.CallMethodWithParmString("\uD83D\uDE80")] Value for json property named value didn't match.]

Expected: str_const_🛶
Actual:   str_const_🚀 
Expected: {
  "type": "string",
  "value": "str_const_🛶"
} 
Actual: {
  "type": "string",
  "value": "str_const_🚀",
  "description": "str_const_🚀"
}
CheckValue failed for this.CallMethodWithParmString("\uD83D\uDE80"). Expected: {
  "type": "string",
  "value": "str_const_🛶"
}, vs {
  "type": "string",
  "value": "str_const_🚀",
  "description": "str_const_🚀"
}
```

Reply:
If you rebase on main and use
```
                    ("this.CallMethodWithParmString(\"🛶\")", TString("str_const_🛶")),
                    ("this.CallMethodWithParmString(\"\\uD83D\\uDEF6\")", TString("str_const_🛶")),
                    ("this.CallMethodWithParmString(\"🚀\")", TString("str_const_🚀")),
```
you should be fine